relay/circuit-v2.md

```diff
@@ -4,7 +4,7 @@ This is the version 2 of the libp2p Circuit Relay protocol.
 
 | Lifecycle Stage | Maturity       | Status | Latest Revision |
 |-----------------|----------------|--------|-----------------|
-| 1A              | DRAFT          | Active | r0, 2021-05-05  |
+| 1A              | DRAFT          | Active | r1, 2021-12-09  |
```

Comment:
I don't think a version bump is necessary, given that this does not change any functionality, i.e. given that implementations do not have to change based on this.

Reply:
Dropped the commit.